gems/sorbet-runtime/lib/types/private/methods/decl_builder.rb

```diff
@@ -167,6 +167,23 @@ def abstract
       self
     end
 
+    def incompatible_override
+      check_live!
```

Comment:
don't copy code, just call `override`
also, did you notice that `override` has `allow_incompatible` arg? I think it was intended to serve the purpose of this method. Can we not have both?

Reply:
I'm removing `incompatible_override` in favor of using `override(allow_incompatible: true)`.